config/travis/runtests.sh

```diff
@@ -10,7 +10,7 @@ set -e;
 
 if test "${TARGET}" = "jenkins";
```

Comment:
maybe change this to jenkins2 and jenkins3 test targets, so both scripts are tested for the intermediate?